state_machine/applications/flight/Tasks/telemetry.py

```diff
@@ -20,15 +19,17 @@ async def main_task(self):
         try:
             self.write_beacon()
         except Exception:
-            files.mkdirp('/sd/logs/beacon/')
+            files.mkdirp('/sd/logs/telemetry/')
             self.write_beacon()
 
     def write_beacon(self):
-        currTime = time.time()
-        TIMEINTERVAL = 1000
-        log_directory = "/sd/logs/"
-        current_file = f"{log_directory}beacon/{int(currTime//TIMEINTERVAL)}.txt"
+        t = cubesat.rtc.datetime
```

Comment:
Follow the same convention as the Tasks/log.py for file naming (`./telemetry/boot_count/time`)